library/src/main/kotlin/it/krzeminski/githubactions/actions/MissingAction.kt

```diff
@@ -0,0 +1,14 @@
+package it.krzeminski.githubactions.actions
+
+/**
+ * Used by the script generator when no Action Wrapper can be found
+ */
+class MissingAction(
```

Comment:
How about
```suggestion
class CustomAction(
```
? MissingAction looks a bit as a placeholder, and it may confuse the user whether such action invocation will actually work or not.